src/status_im/ui/components/svgimage.cljs

```diff
@@ -0,0 +1,43 @@
+(ns status-im.ui.components.svgimage
+  (:require [status-im.ui.components.react :as react]
+            [reagent.core :as reagent]))
+
+(defn html [uri width height]
+  (str
+   "<!DOCTYPE html>\n
+   <html>
+   <head>
+   <style type=\"text/css\">
+   img {
+        width: 100%;
+        height: 100%;
+        margin: 0 auto;
+        }
+
+   div {
+        width: " (if width (str width "px") "auto")
+   "height: " (if height (str height "px") "auto")
+
+   "}
+    body {margin: 0;}
+
+    </style>
+    </head>
+    <body>
+    <div>
+    <img src=" uri " align=\"middle\" />
+    </div>
+    </body>
+    </html>"))
+
+(defn svgimage [{:keys [style source]}]
```

Comment:
Have you thought making this a part of the image component? So that if the source is svg use this, if no use regular image?
This way it would be transparent for image users.

Reply:
yes i thought, but decided to do not do it, because it's a temporary solution, and should be used only for collectables